package/MDAnalysis/analysis/base.py

```diff
@@ -92,7 +92,7 @@ def _conclude(self):
         """
         pass
 
-    def run(self):
+    def run(self, **kwargs):
```

Comment:
I think the rationale was that run functions are always happy to take in kwars... but I think I can be easily swayed to say that we should only advertise what one particular run function needs (and I know that you're a proponent of *explicit kwargs*). So, yes, I come around to the idea to only put in those args that are really needed.